concent_api/core/views.py

```diff
@@ -99,13 +99,16 @@ def receive(request, _message):
     decoded_message_data = deserialize_message(last_undelivered_message_status.message.data.tobytes())
 
     assert last_undelivered_message_status.message.type == decoded_message_data.TYPE
-
     if last_undelivered_message_status.message.type == message.ForceReportComputedTask.TYPE:
         if client_public_key != last_undelivered_message_status.message.auth.requestor_public_key_bytes:
             return None
-        if decoded_message_data.task_to_compute.compute_task_def['deadline'] + settings.CONCENT_MESSAGING_TIME < current_time:
+        if decoded_message_data.report_computed_task.task_to_compute.compute_task_def['deadline'] + settings.CONCENT_MESSAGING_TIME < current_time:
             set_message_as_delivered(last_undelivered_message_status)
-            return handle_receive_ack_from_force_report_computed_task(decoded_message_data)
+            return handle_receive_ack_from_force_report_computed_task(
+                request,
+                decoded_message_data,
+                last_undelivered_message_status.message.auth
```

Comment:
I think that we should keep consistency here. With this changes we pass `.auth` in some functions and `last_undelivered_message_status` in others. I think we should, if its possible, try to keep it the same for all functions on the same level. @cameel What do you think ?